app/assets/v2/js/notifications.js

```diff
@@ -18,9 +18,14 @@ Vue.mixin({
 
       $.when(getNotifications).then(function(response) {
         newNotifications = newData(response.data, vm.notifications);
-        newNotifications.forEach(function(item) {
-          vm.notifications.push(item);
-        });
+
+        if (vm.notifications.length === 0) {
+          newNotifications.forEach(function(item) {
+            vm.notifications.push(item);
+          });
+        } else if (newNotifications.length > 0) {
```

Comment:
not sure why this change, this will add the notification to the end
![image](https://user-images.githubusercontent.com/1358093/75054026-0949d680-54b1-11ea-8f17-690465a5c4df.png)

Reply:
It's strange. I just tested the changes to avoid this behavior. 😱

Reply:
That is because if you reload the page then the result will be on the top, but if you click the notifications to open/close, that will fetch  new notifications and then will be attached to the end.

Reply:
just tested the new changes and now is getting on the bottom, maybe when I tested my js was cached not sure what changed because I don't remember this on the past. Maybe you are right we will need to do concat or unshift

Reply:
ok, dont worry about this part I will do a hotfix on stable about this order but not here, in the filter part

Reply:
fixed 68777da012cea06c64105dfa2dea77bfb01bce6e

Reply:
Thanks for the fix @octavioamu 👍